### ISSUE TITLE
2024-12-13, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#19]
# Today's Arxiv Papers

(1) [EasyRef: Omni-Generalized Group Image Reference for Diffusion Models via Multimodal LLM](http://arxiv.org/abs/2412.09618v1), 2024-12-12

(2) [Olympus: A Universal Task Router for Computer Vision Tasks](http://arxiv.org/abs/2412.09612v1), 2024-12-12

(3) [SynerGen-VL: Towards Synergistic Image Understanding and Generation with Vision Experts and Token Folding](http://arxiv.org/abs/2412.09604v1), 2024-12-12

(4) [Do Multimodal Large Language Models See Like Humans?](http://arxiv.org/abs/2412.09603v1), 2024-12-12

(5) [TimeRefine: Temporal Grounding with Time Refining Video LLM](http://arxiv.org/abs/2412.09601v1), 2024-12-12

(6) [InternLM-XComposer2.5-OmniLive: A Comprehensive Multimodal System for Long-term Streaming Video and Audio Interactions](http://arxiv.org/abs/2412.09596v1), 2024-12-12

(7) [OLA-VLM: Elevating Visual Perception in Multimodal LLMs with Auxiliary Embedding Distillation](http://arxiv.org/abs/2412.09585v1), 2024-12-12

(8) [Neptune: The Long Orbit to Benchmarking Long Video Understanding](http://arxiv.org/abs/2412.09582v1), 2024-12-12

(9) [DiverseAgentEntropy: Quantifying Black-Box LLM Uncertainty through Diverse Perspectives and Multi-Agent Interaction](http://arxiv.org/abs/2412.09572v1), 2024-12-12

(10) [JuStRank: Benchmarking LLM Judges for System Ranking](http://arxiv.org/abs/2412.09569v1), 2024-12-12

(11) [Obfuscated Activations Bypass LLM Latent-Space Defenses](http://arxiv.org/abs/2412.09565v1), 2024-12-12

(12) [Does Representation Matter? Exploring Intermediate Layers in Large Language Models](http://arxiv.org/abs/2412.09563v1), 2024-12-12

(13) [Foundational Large Language Models for Materials Research](http://arxiv.org/abs/2412.09560v1), 2024-12-12

(14) [Exemplar Masking for Multimodal Incremental Learning](http://arxiv.org/abs/2412.09549v1), 2024-12-12

(15) [Can Modern LLMs Act as Agent Cores in Radiology~Environments?](http://arxiv.org/abs/2412.09529v1), 2024-12-12

(16) [Lyra: An Efficient and Speech-Centric Framework for Omni-Cognition](http://arxiv.org/abs/2412.09501v1), 2024-12-12

(17) [The Impact of Copyrighted Material on Large Language Models: A Norwegian Perspective](http://arxiv.org/abs/2412.09460v1), 2024-12-12

(18) [ATPrompt: Textual Prompt Learning with Embedded Attributes](http://arxiv.org/abs/2412.09442v1), 2024-12-12

(19) [From Intention To Implementation: Automating Biomedical Research via LLMs](http://arxiv.org/abs/2412.09429v1), 2024-12-12

(20) [Unifying AI Tutor Evaluation: An Evaluation Taxonomy for Pedagogical Ability Assessment of LLM-Powered AI Tutors](http://arxiv.org/abs/2412.09416v1), 2024-12-12

(21) [Imitate, Explore, and Self-Improve: A Reproduction Report on Slow-thinking Reasoning Systems](http://arxiv.org/abs/2412.09413v1), 2024-12-12

(22) [LLMs for Time Series: an Application for Single Stocks and Statistical Arbitrage](http://arxiv.org/abs/2412.09394v1), 2024-12-12

(23) [AI Predicts AGI: Leveraging AGI Forecasting and Peer Review to Explore LLMs' Complex Reasoning Capabilities](http://arxiv.org/abs/2412.09385v1), 2024-12-12

(24) [Delving into Youth Perspectives on In-game Gambling-like Elements: A Proof-of-Concept Study Utilising Large Language Models for Analysing User-Generated Text Data](http://arxiv.org/abs/2412.09345v1), 2024-12-12

(25) [Benchmarking LLMs for Mimicking Child-Caregiver Language in Interaction](http://arxiv.org/abs/2412.09318v1), 2024-12-12

(26) [CRVQ: Channel-relaxed Vector Quantization for Extreme Compression of LLMs](http://arxiv.org/abs/2412.09282v1), 2024-12-12

(27) [Learning to Solve Domain-Specific Calculation Problems with Knowledge-Intensive Programs Generator](http://arxiv.org/abs/2412.09280v1), 2024-12-12

(28) [Towards a Multimodal Large Language Model with Pixel-Level Insight for Biomedicine](http://arxiv.org/abs/2412.09278v1), 2024-12-12

(29) [Towards Understanding the Robustness of LLM-based Evaluations under Perturbations](http://arxiv.org/abs/2412.09269v1), 2024-12-12

(30) [GeLoRA: Geometric Adaptive Ranks For Efficient LoRA Fine-tuning](http://arxiv.org/abs/2412.09250v1), 2024-12-12

(31) [A Systematic Review of Knowledge Tracing and Large Language Models in Education: Opportunities, Issues, and Future Research](http://arxiv.org/abs/2412.09248v1), 2024-12-12

(32) [Make Satire Boring Again: Reducing Stylistic Bias of Satirical Corpus by Utilizing Generative LLMs](http://arxiv.org/abs/2412.09247v1), 2024-12-12

(33) [SPRec: Leveraging Self-Play to Debias Preference Alignment for Large Language Model-based Recommendations](http://arxiv.org/abs/2412.09243v1), 2024-12-12

(34) [LMAgent: A Large-scale Multimodal Agents Society for Multi-user Simulation](http://arxiv.org/abs/2412.09237v1), 2024-12-12

(35) [LIVE-GS: LLM Powers Interactive VR by Enhancing Gaussian Splatting](http://arxiv.org/abs/2412.09176v1), 2024-12-12

(36) [ReFF: Reinforcing Format Faithfulness in Language Models across Varied Tasks](http://arxiv.org/abs/2412.09173v1), 2024-12-12

(37) [When Text Embedding Meets Large Language Model: A Comprehensive Survey](http://arxiv.org/abs/2412.09165v1), 2024-12-12

(38) [Filter-then-Generate: Large Language Models with Structure-Text Adapter for Knowledge Graph Completion](http://arxiv.org/abs/2412.09094v1), 2024-12-12

(39) [Forest-of-Thought: Scaling Test-Time Compute for Enhancing LLM Reasoning](http://arxiv.org/abs/2412.09078v1), 2024-12-12

(40) [EmbedGenius: Towards Automated Software Development for Generic Embedded IoT Systems](http://arxiv.org/abs/2412.09058v1), 2024-12-12

(41) [Dial-In LLM: Human-Aligned Dialogue Intent Clustering with LLM-in-the-loop](http://arxiv.org/abs/2412.09049v1), 2024-12-12

(42) [Oversight in Action: Experiences with Instructor-Moderated LLM Responses in an Online Discussion Forum](http://arxiv.org/abs/2412.09048v1), 2024-12-12

(43) [Multi-Task Learning with LLMs for Implicit Sentiment Analysis: Data-level and Task-level Automatic Weight Learning](http://arxiv.org/abs/2412.09046v1), 2024-12-12

(44) [ZigZagkv: Dynamic KV Cache Compression for Long-context Modeling based on Layer Uncertainty](http://arxiv.org/abs/2412.09036v1), 2024-12-12

(45) [What Makes Cryptic Crosswords Challenging for LLMs?](http://arxiv.org/abs/2412.09012v1), 2024-12-12

(46) [Assessing the Robustness of Retrieval-Augmented Generation Systems in K-12 Educational Question Answering with Knowledge Discrepancies](http://arxiv.org/abs/2412.08985v1), 2024-12-12

(47) [RuleArena: A Benchmark for Rule-Guided Reasoning with LLMs in Real-World Scenarios](http://arxiv.org/abs/2412.08972v1), 2024-12-12

(48) [Reasoning-Aware Query-Focused Summarization over Multi-Table Data](http://arxiv.org/abs/2412.08970v1), 2024-12-12

(49) [Align, Generate, Learn: A Novel Closed-Loop Framework for Cross-Lingual In-Context Learning](http://arxiv.org/abs/2412.08955v1), 2024-12-12

(50) [AI-assisted Knowledge Discovery in Biomedical Literature to Support Decision-making in Precision Oncology](http://arxiv.org/abs/2412.08900v1), 2024-12-12

(51) [Neural Interactive Proofs](http://arxiv.org/abs/2412.08897v1), 2024-12-12

(52) [Lexico: Extreme KV Cache Compression via Sparse Coding over Universal Dictionaries](http://arxiv.org/abs/2412.08890v1), 2024-12-12

(53) [A Graph-Based Synthetic Data Pipeline for Scaling High-Quality Reasoning Instructions](http://arxiv.org/abs/2412.08864v1), 2024-12-12

(54) [MOPI-HFRS: A Multi-objective Personalized Health-aware Food Recommendation System with LLM-enhanced Interpretation](http://arxiv.org/abs/2412.08847v1), 2024-12-12

(55) [Exploring Large Language Models on Cross-Cultural Values in Connection with Training Methodology](http://arxiv.org/abs/2412.08846v1), 2024-12-12

(56) [Kajal: Extracting Grammar of a Source Code Using Large Language Models](http://arxiv.org/abs/2412.08842v1), 2024-12-12

(57) [Large Concept Models: Language Modeling in a Sentence Representation Space](http://arxiv.org/abs/2412.08821v1), 2024-12-11

(58) [HARP: A challenging human-annotated math reasoning benchmark](http://arxiv.org/abs/2412.08819v1), 2024-12-11

(59) [Autoformalizing and Simulating Game-Theoretic Scenarios using LLM-augmented Agents](http://arxiv.org/abs/2412.08805v1), 2024-12-11

(60) [Coverage-based Fairness in Multi-document Summarization](http://arxiv.org/abs/2412.08795v1), 2024-12-11

(61) [LLaVA-Zip: Adaptive Visual Token Compression with Intrinsic Image Information](http://arxiv.org/abs/2412.08771v1), 2024-12-11

(62) [In-Context Learning with Topological Information for Knowledge Graph Completion](http://arxiv.org/abs/2412.08742v1), 2024-12-11

(63) [Euclid: Supercharging Multimodal LLMs with Synthetic High-Fidelity Visual Descriptions](http://arxiv.org/abs/2412.08737v1), 2024-12-11

(64) [Generative Semantic Communication: Architectures, Technologies, and Applications](http://arxiv.org/abs/2412.08642v1), 2024-12-11

(65) [LatentQA: Teaching LLMs to Decode Activations Into Natural Language](http://arxiv.org/abs/2412.08686v1), 2024-12-11

(66) [ChatDyn: Language-Driven Multi-Actor Dynamics Generation in Street Scenes](http://arxiv.org/abs/2412.08685v1), 2024-12-11

(67) [Fast Prompt Alignment for Text-to-Image Generation](http://arxiv.org/abs/2412.08639v1), 2024-12-11

(68) [Multimodal Latent Language Modeling with Next-Token Diffusion](http://arxiv.org/abs/2412.08635v1), 2024-12-11

(69) [Synthetic Vision: Training Vision-Language Models to Understand Physics](http://arxiv.org/abs/2412.08619v1), 2024-12-11

(70) [Exploiting the Index Gradients for Optimization-Based Jailbreaking on Large Language Models](http://arxiv.org/abs/2412.08615v1), 2024-12-11

(71) [Preference Discerning with LLM-Enhanced Generative Retrieval](http://arxiv.org/abs/2412.08604v1), 2024-12-11

(72) [Empirical Measurements of AI Training Power Demand on a GPU-Accelerated Node](http://arxiv.org/abs/2412.08602v1), 2024-12-11

(73) [Leveraging Graph-RAG and Prompt Engineering to Enhance LLM-Based Automated Requirement Traceability and Compliance Checks](http://arxiv.org/abs/2412.08593v1), 2024-12-11

(74) [Distinguishing Scams and Fraud with Ensemble Learning](http://arxiv.org/abs/2412.08680v1), 2024-12-11

(75) [Advancing Single- and Multi-task Text Classification through Large Language Model Fine-tuning](http://arxiv.org/abs/2412.08587v1), 2024-12-11

(76) [TURBOATTENTION: Efficient Attention Approximation For High Throughputs LLMs](http://arxiv.org/abs/2412.08585v1), 2024-12-11

(77) [Automated Soap Opera Testing Directed by LLMs and Scenario Knowledge: Feasibility, Challenges, and Road Ahead](http://arxiv.org/abs/2412.08581v1), 2024-12-11

(78) [Can We Generate Visual Programs Without Prompting LLMs?](http://arxiv.org/abs/2412.08564v1), 2024-12-11

(79) [Underestimated Privacy Risks for Minority Populations in Large Language Model Unlearning](http://arxiv.org/abs/2412.08559v1), 2024-12-11

(80) [MaestroMotif: Skill Design from Artificial Intelligence Feedback](http://arxiv.org/abs/2412.08542v1), 2024-12-11

(81) [EMS: Adaptive Evict-then-Merge Strategy for Head-wise KV Cache Compression Based on Global-Local Importance](http://arxiv.org/abs/2412.08521v1), 2024-12-11

(82) [Bridging Relevance and Reasoning: Rationale Distillation in Retrieval-Augmented Generation](http://arxiv.org/abs/2412.08519v1), 2024-12-11

(83) [AltFS: Agency-light Feature Selection with Large Language Models in Deep Recommender Systems](http://arxiv.org/abs/2412.08516v1), 2024-12-11

(84) [Strategies and Challenges of Efficient White-Box Training for Human Activity Recognition](http://arxiv.org/abs/2412.08507v1), 2024-12-11

(85) [SuperCode: Sustainability PER AI-driven CO-DEsign](http://arxiv.org/abs/2412.08490v1), 2024-12-11

(86) [Multi-GraspLLM: A Multimodal LLM for Multi-Hand Semantic Guided Grasp Generation](http://arxiv.org/abs/2412.08468v1), 2024-12-11

(87) [TapeAgents: a Holistic Framework for Agent Development and Optimization](http://arxiv.org/abs/2412.08445v1), 2024-12-11

(88) [From Multimodal LLMs to Generalist Embodied Agents: Methods and Lessons](http://arxiv.org/abs/2412.08442v1), 2024-12-11

(89) [Assessing Personalized AI Mentoring with Large Language Models in the Computing Field](http://arxiv.org/abs/2412.08430v1), 2024-12-11

(90) [SwarmGPT-Primitive: A Language-Driven Choreographer for Drone Swarms Using Safe Motion Primitive Composition](http://arxiv.org/abs/2412.08428v1), 2024-12-11

(91) [Detecting Conversational Mental Manipulation with Intent-Aware Prompting](http://arxiv.org/abs/2412.08414v1), 2024-12-11

(92) [Learning to Reason via Self-Iterative Process Feedback for Small Language Models](http://arxiv.org/abs/2412.08393v1), 2024-12-11

(93) [SweetieChat: A Strategy-Enhanced Role-playing Framework for Diverse Scenarios Handling Emotional Support Agent](http://arxiv.org/abs/2412.08389v1), 2024-12-11

(94) [NyayaAnumana & INLegalLlama: The Largest Indian Legal Judgment Prediction Dataset and Specialized Language Model for Enhanced Decision Analysis](http://arxiv.org/abs/2412.08385v1), 2024-12-11

(95) [HyViLM: Enhancing Fine-Grained Recognition with a Hybrid Encoder for Vision-Language Models](http://arxiv.org/abs/2412.08378v1), 2024-12-11

(96) [SmolTulu: Higher Learning Rate to Batch Size Ratios Can Lead to Better Reasoning in SLMs](http://arxiv.org/abs/2412.08347v1), 2024-12-11

(97) [Large Language Models Still Face Challenges in Multi-Hop Reasoning with External Knowledge](http://arxiv.org/abs/2412.08317v1), 2024-12-11

(98) [Code LLMs: A Taxonomy-based Survey](http://arxiv.org/abs/2412.08291v1), 2024-12-11

(99) [Lachesis: Predicting LLM Inference Accuracy using Structural Properties of Reasoning Paths](http://arxiv.org/abs/2412.08281v1), 2024-12-11

(100) [Y-NQ: English-Yorùbá Evaluation dataset for Open-Book Reading Comprehension and Text Generation](http://arxiv.org/abs/2412.08279v1), 2024-12-11
